src/main/java/net/atos/zac/app/mail/MailRestService.java

```diff
@@ -73,7 +73,7 @@ public void sendMail(
             final RESTMailGegevens restMailGegevens
     ) {
         final Zaak zaak = zrcClientService.readZaak(zaakUUID);
-        assertPolicy(policyService.readZaakRechten(zaak).versturenEmail());
+        assertPolicy(policyService.readZaakRechten(zaak).versturenEmailBijAfhandelenZaak());
```

Comment:
Did you check that this REST endpoint is only used in the scenario 'send email when closing zaak'? If it is I think we should renamed this REST endpoint to make that clear. If it is not this policy check is probably not ok here?

Reply:
I mean: it is now a generic endpoint to send email so having this policy check is bound to cause confusion for future developers. It is really weird.

Reply:
Yes it is used. It's the only endpoint for sending mail I think, so used for most mail related tasks.

We'll have to create another endpoint and change UI to use it.

Reply:
If that's the case the current PR as it is actually breaks functionality then maybe?

Reply:
This PR will just restore the versturenEmail policy functionality